src/main/java/tr/ogu/edu/school/schoolapp/controller/ActTicketController.java

```diff
@@ -41,10 +40,13 @@ public ResponseEntity<Resource> getTicketAsPdf(@PathVariable String verification
 	}
 
 	// Biletin kullanım durumu
-	@PostMapping("/verify")
-    	public ResponseEntity<Map<String, Object>> verifyTicket(@RequestBody String qrCode) {
-		// Biletin durumunu doğrular ve günceller
-		Map<String, Object> response = actTicketService.verifyTicket(qrCode);
-		return ResponseEntity.ok(response);
+	@PostMapping("/public/{verificationCode}/{line}/{no}")
+	public ResponseEntity<Map<String, Object>> verifyTicket(@PathVariable String verificationCode,
+        	@PathVariable String line, @PathVariable String no) {
+
+    	// Biletin durumunu doğrular ve günceller
+    	String qrCode = verificationCode + "/" + line + "/" + no;
```

Comment:
neden line ve noyu verification koda ekledik. line ve no parametre olarak geliyor kullanmak zorunda değilsin ya da belki sadece doğrulama için kullanabilirsin.